packages/generator-one-app-module/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+The following guidelines must be followed by all contributors to this repository. Please review them carefully and do not hesitate to ask for help.
+
+### Code of Conduct
+
+* Review and test your code before submitting a pull request.
+* Be kind and professional. Avoid assumptions; oversights happen.
+* Be clear and concise when documenting code; focus on value.
+* Don't commit commented code to the main repo (stash locally, if needed).
+
+See [our code of conduct](./CODE_OF_CONDUCT) for more details.
```

Comment:
This is a broken link as is, the CODE_OF_CONDUCT file is in the top level readme.

Reply:
removed `CONTRIBUTING.md` alltogether since it's at the top-level and most of the content was redundant. Plus no other packages had it.

Reply:
generator-specific contributing guidelines are now housed within `README.md`